main.c

```diff
@@ -83,6 +83,8 @@ struct _uintptr_to_must_hold_fuse_ino_t_dummy_struct
 };
 #endif
 
+static bool disable_ovl_whiteout;
```

Comment:
Should this be initialized?

Reply:
non initialized global variables end in the `.bss` section.  It will be initialized to zero when the program starts